### PR TITLE
Checkbox name

### LINKED ISSUE #1107: Radio and Checkbox patterns
Curious about the difference in pattern between radio and checkboxes.

Radio

```
 <fieldset class="usa-fieldset-inputs usa-sans">
    <legend class="usa-sr-only">Historical figures 2</legend>
    <ul class="usa-unstyled-list">
      <li>
        <input id="pea-soup" type="radio" checked="" name="soup" value="pea">
        <label for="pea-soup">Elizabeth Cady Stanton</label>
      </li>
      <li>
        <input id="chicken-noodle" type="radio" name="soup" value="chicken-noodle">
        <label for="chicken-noodle">Susan B. Anthony</label>
      </li>
      <li>
        <input id="tomato" type="radio" name="soup" value="tomato">
        <label for="tomato">Harriet Tubman</label>
      </li>
    </ul>
  </fieldset>
```

Checkbox 

```
  <fieldset class="usa-fieldset-inputs usa-sans">
    <legend class="usa-sr-only">Historical figures 1</legend>
    <ul class="usa-unstyled-list">
      <li>
        <input id="apple-pie" type="checkbox" name="apple-pie" value="apple-pie" checked="">
        <label for="apple-pie">Sojourner Truth</label>
      </li>
      <li>
        <input id="key-lime-pie" type="checkbox" name="key-lime-pie" value="key-lime-pie">
        <label for="key-lime-pie">Frederick Douglass</label>
      </li>
      <li>
        <input id="peach-pie" type="checkbox" name="peach-pie" value="peach-pie">
        <label for="peach-pie">Booker T. Washington</label>
      </li>
      <li>
        <input id="disabled" type="checkbox" disabled="">
        <label for="disabled">George Washington Carver</label>
      </li>
    </ul>
  </fieldset>
```

From the checkbox code above, it would be necessary to check for an "is set" equivalent on each checkbox possibility. Alternatively, we could follow the radio pattern - using name - and have it POST as an array. (This is usually the pattern seen in PHP or Ruby.)

Checkbox alt

```
  <fieldset class="usa-fieldset-inputs usa-sans">
    <legend class="usa-sr-only">Historical figures 1</legend>
    <ul class="usa-unstyled-list">
      <li>
        <input id="apple-pie" type="checkbox" name="pie[]" value="apple-pie" checked="">
        <label for="apple-pie">Sojourner Truth</label>
      </li>
      <li>
        <input id="key-lime-pie" type="checkbox" name="pie[]" value="key-lime-pie">
        <label for="key-lime-pie">Frederick Douglass</label>
      </li>
      <li>
        <input id="peach-pie" type="checkbox" name="pie[]" value="peach-pie">
        <label for="peach-pie">Booker T. Washington</label>
      </li>
      <li>
        <input id="disabled" type="checkbox" disabled="">
        <label for="disabled">George Washington Carver</label>
      </li>
    </ul>
  </fieldset>
```

This allows the posting of an array of the check values - then we only have to check for the "pie" variable in the POST data. Again, curious if this was a conscious decision to move a different way. Otherwise, could be an issue.

### PR DESCRIPTION
**Pull request by jbrucegsa**
Fixes #1107

### CODE REVIEW

File: docs/pages/getting-started.html [hunk 1]
```diff
@@ -52,6 +52,10 @@ <h3 class="usa-heading">For developers</h3>
...
 
+<h4>Progressive Enhancement:</h4>
+
+<p>The standards embrace Progressive Enhancement, which is a web development strategy with accessibility and responsiveness at its heart. Server-side languages and frameworks are leveraged for the majority of <strong>HTML</strong> generation. These pages are enhanced with CSS to create a rich user experience and design. Finally, client-side languages (like JavaScript) are applied to add the final layer of enhancement to the experience without losing the underlying functionality of the page or site. This means the majority of processes are run on the servers hosting the site, therefore, not dependent on resource availability of the client (mobile devices typically have fewer resources than a desktop or laptop, for example). Further, you can focus on the functionality of the site, ensuring it works properly (the expected HTML is delivered) before continuing to enhance the experience. Finally, by maximizing server-side processing, you minimize the negative impact on battery life client-side processing can have.</p>
+
 <h3 class="usa-heading">For designers</h3>
```
Comment (maya) [inline docs/pages/getting-started.html]: This shouldn't be included in the PR.

**Comment by maya** [comment]
@jbrucegsa thanks for submitting! any chance you can submit off of a fresh pull from `18f-pages-staging`. This seems to have gotten a lot of other commits stuck to it.

**Comment by jbrucegsa** [comment]
@maya - See other PR.
